web/impact/impact/graphql/query.py

```diff
@@ -25,7 +25,8 @@ class Query(graphene.ObjectType):
         EntrepreneurProfileType, id=graphene.Int())
 
     def resolve_expert_profile(self, info, **kwargs):
-        user_id = kwargs.get('id')
+        user_id = kwargs.get('id') or info.context.user.id
```

Comment:
use the default option for kwargs.get:

`user_id = kwargs.get('id', info.context.user.id)`